test/modelmanagers/usermodel.test.js

```diff
@@ -0,0 +1,31 @@
+const chai = require('chai');
+const userModelManager = require('../../lib/modelManagers/usermodel');
+
+const { expect } = chai;
+
+const email = 'testmail@yahoo.com',
+  password = '111111',
+  firstname = 'testfirstname',
+  lastname = 'testlastname';
+
+let createdUser;
+
+describe('Test for sending user info into database', () => {
+  it('Should have method that sends user info to database', () => {
+    expect(userModelManager.create).to.be.a('function');
+  });
+
+  it('should return a creted user object', async () => {
+    before(() => {
+      expect(firstname).to.be.a('string');
```

Comment:
The `before` function is not met for assertions.

Reply:
Thanks for that info. I have made the changes